cranelift/codegen/src/isa/arm32/mod.rs

```diff
@@ -92,6 +92,14 @@ impl MachBackend for Arm32Backend {
         &self.flags
     }
 
+    fn enabled_isa_flags(&self) -> Vec<String> {
+        Vec::new()
+    }
+
+    fn is_flag_enabled(&self, _flag: &str) -> bool {
+        false
+    }
+
     fn hash_all_flags(&self, mut hasher: &mut dyn Hasher) {
```

Comment:
FWIW this method can probably be removed since we can just hash the return value of `enabled_isa_flags` I think?

Reply:
This is still used for the compiler's hash implementation to generate the unique-for-this-system hash used by the code cache.

Given that implementation is based on `Hash`, I felt it easier to just keep as-is than implement something else that could do the less-specific checks that deserializing the AOT artifact now does.